core/src/main/java/org/axonframework/eventhandling/EventProcessor.java

```diff
@@ -59,6 +61,13 @@
      */
     Registration registerInterceptor(MessageHandlerInterceptor<? super EventMessage<?>> interceptor);
 
+    /**
+     * Return the list of registered interceptors for this event processor
+     *
+     * @return the list of interceptors of this event processor
+     */
+    Set<MessageHandlerInterceptor<? super EventMessage<?>>> getInterceptors();
```

Comment:
So I placed the comment on the `AbstractEventProcessor` as well, but it extends to this interface of course.
Please add `Handler` to the function name.